### PR TITLE
Creating FeedbackCards

### PR DESCRIPTION
**Pull request by mrdedede**
- Created FeedbackCards for app
- Made it iterable

### CODE REVIEW

File: Foundations/Pages/Feedbacks.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        FeedBackEntry(user: "Lisa Mello", image: "lisa-mello", comment: "Parabéns! Vi que você vem trabalhando há tempos para construir essa conquista. Na minha opinião o que te levou até ela foi sua determinação e como você lida com erros, e na sua?"),
+        FeedBackEntry(user: "Kiev Gama", image: "kiev-gama", comment: "Parabéns! Vi que você vem trabalhando há tempos para construir essa conquista. Na minha opinião o que te levou até ela foi sua determinação e como você lida com erros, e na sua?")
+        ]
+    
+    var body: some View {
+        VStack (spacing: 10) {
+            Text("Feedbacks")
+                .font(.title)
+            ForEach(sampleData, id: \.self) { datum in
+                VStack (spacing: 10) {
```
Comment (JDaniloC) [inline Foundations/Pages/Feedbacks.swift]: A VStack with a single element? If you want to add a spacing, use padding in the HStack.

**Review by JDaniloC — CHANGES_REQUESTED** [review]
I need to test

File: Foundations/Pages/Feedbacks.swift (hunk 1 above)
Comment (mrdedede) [reply Foundations/Pages/Feedbacks.swift]: Done

**Review by JDaniloC — APPROVED** [review]
I think is OK.

**Review by Jailsonsdsj — APPROVED** [review]

**Pull request merged**